src/custodia/client.py

```diff
@@ -12,6 +12,7 @@
 from requests.adapters import HTTPAdapter
 from requests.compat import unquote, urlparse
 
+# pylint: disable=import-error
```

Comment:
I saw this case: https://github.com/latchset/custodia/blob/master/src/custodia/httpd/server.py#L20
and I thought that was something similar here.